dds/idl/dds_generator.h

```diff
@@ -564,7 +558,7 @@ void generateCaseBody(
   CommonFn commonFn, CommonFn commonFn2, AST_UnionBranch* branch,
   const char* statementPrefix, const char* namePrefix,
   const char* uni, bool generateBreaks, bool parens,
-  bool printing = false, unsigned default_id = 0)
+  bool printing = false, unsigned default_id = 0, AST_Structure* type = NULL)
```

Comment:
```suggestion
  bool printing = false, unsigned default_id = 0, AST_Structure* type = 0)
```

Reply:
Why would 0 be better for the pointer default value, than NULL?

Reply:
It is not necessarily "better," but it is what we use:

https://github.com/objectcomputing/OpenDDS/blob/master/docs/guidelines.md#pointers-and-references